src/lib/ui/ExtensionsDetailsBottom.svelte

```diff
@@ -0,0 +1,58 @@
+<script lang="ts">
+import moment from 'moment';
+
+import type { CatalogExtensionInfo, CatalogExtensionVersionInfo } from '$lib/api/extensions-info';
+import Markdown from '$lib/Markdown.svelte';
+
+const { extension }: { extension: CatalogExtensionInfo } = $props();
+
+// hide preview versions
+const versions: CatalogExtensionVersionInfo[] = $derived(extension.versions?.filter(v => v.preview === false));
+
+let readmeContent: string | undefined = $state(undefined);
+
+$effect(() => {
+  const fetchReadme = async (): Promise<void> => {
+    const readmeUrl = extension.versions?.[0]?.files.find(file => file.assetType === 'README')?.data;
+    if (!readmeUrl) {
+      return;
+    }
+    const response = await fetch(readmeUrl);
+    const data = await response.text();
+    readmeContent = data;
+  };
+
+  fetchReadme().catch((e: unknown) => {
+    console.error(e);
+  });
+});
+</script>
+
+<div class="w-full min-h-full overflow-y-visible leading-6 text-[var(--pd-details-body-text)] p-4">
+	<div class="bg-[var(--pd-invert-content-card-bg)] p-4 rounded">
+		{#if readmeContent}
```

Comment:
note: If there is no readme, then it is always going to show `Loading...`. Showing a message `readme is not available` would be more accurate IMO.

Reply:
the readme is mandatory